elements/rh-accordion/rh-accordion.css

```diff
@@ -0,0 +1,35 @@
+:host {
+  --_border-color: var(--rh-color-border-subtle-on-light, #d2d2d2);
+
+  color: var(--rh-color-black-900, #151515);
+  background-color: var(--rh-color-white, #ffffff);
+}
+
+:host([on="dark"]) {
+  --_border-color: var(--rh-color-black-600, #6a6e73);
+}
+#container { display: contents; }
+
+::slotted(rh-accordion-header:first-child) {
+  display: block;
+  border-top: 1px solid var(--_border-color);
+  border-bottom: 1px solid var(--_border-color);
+}
+
+::slotted(rh-accordion-header:not(:first-child)) {
+  display: block;
+  border-bottom: 1px solid var(--_border-color);
+}
+
+::slotted(rh-accordion-header:is([expanded])) {
+  display: block;
+  border-bottom: 0;
+  box-shadow: 0 2px 4px #15151533;
```

Comment:
Should this be using a token value?  I do notice there is a alpha difference 33% vs 20% in the token.

```Suggestion
  box-shadow: var(--rh-box-shadow-sm, 0 2px 4px rgba(21, 21, 21, 0.2));
```